code/2_analysis/01_make_maps.R

```diff
@@ -0,0 +1,135 @@
+# Description: This file produces all maps in the paper. Thus, it generates
+# Figures 1, 2a, 2c, 2e, 3a, and Appendix Figures A1a, A1c, A4a and A4c.
+
+# ENVIRONMENT SETUP ------------------------------------------------------------
+
+# run setup script - delete setwd() line when using bash script
+setwd("/win/l1werpfile2/shared/Cheikh/Research/Ishan/Water/pp_replication")
```

Comment:
@cheikhfall07 I think this `setwd` should be removed?

Reply:
solved!